packages/next/src/cli/next-dev.ts

```diff
@@ -19,14 +19,15 @@ import { PHASE_DEVELOPMENT_SERVER } from '../shared/lib/constants'
 import path from 'path'
 import type { NextConfigComplete } from '../server/config-shared'
 import { traceGlobals } from '../trace/shared'
-import { Telemetry } from '../telemetry/storage'
-import loadConfig from '../server/config'
-import { findPagesDir } from '../lib/find-pages-dir'
-import { fileExists, FileType } from '../lib/file-exists'
-import { getNpxCommand } from '../lib/helpers/get-npx-command'
-import { createSelfSignedCertificate } from '../lib/mkcert'
+// Heavy modules - lazy loaded to speed up CLI startup
+// import { Telemetry } from '../telemetry/storage' - loaded in handleSessionStop
```

Comment:
some of those are useful to lazy load, but not all of them, since the telemetry one is needed at all times either way. i need to clean up those comments.